cheatsheets/Password_Storage_Cheat_Sheet.md

```diff
@@ -11,7 +11,7 @@ This cheat sheet provides guidance on the various areas that need to be consider
 - **Use [Argon2id](#argon2id) with a minimum configuration of 15 MiB of memory, an iteration count of 2, and 1 degree of parallelism.**
 - **If [Argon2id](#argon2id) is not available, use [scrypt](#scrypt) with a minimum CPU/memory cost parameter of (2^16), a minimum block size of 8 (1024 bytes), and a parallelization parameter of 1.**
 - **For legacy systems using [bcrypt](#bcrypt), use a work factor of 10 or more and with a password limit of 72 bytes.**
-- **If FIPS-140 compliance is required, use [PBKDF2](#pbkdf2) with a work factor of 310,000 or more and set with an internal hash function of HMAC-SHA-256.**
+- **If FIPS-140 compliance is required, use [PBKDF2](#pbkdf2) with a work factor of 600,000 or more and set with an internal hash function of HMAC-SHA-256.**
```

Comment:
Just a suggestion, but it may be a good idea to provide a link to the specific NIST FIPS-140 and other references, including section #s when appropriate. Especially in this case of NIST FIPS-140, where the cost factor has almost doubled.  Bumping it from 310k to 600k could impact a _lot_ of infrastructure, so many might find this figure hard to believe (especially those who end up footing the bill for the additional CPU resources).  This is partly because other (non-FIPS) NIST documents such as [NIST SP 800-132](https://nvlpubs.nist.gov/nistpubs/Legacy/SP/nistspecialpublication800-132.pdf) or [NIST SP 800-63B](https://pages.nist.gov/800-63-3/sp800-63b.html) have different PBKDF2 cost factors (i.e., iteration counts) recommended. If we allow people to rely only on internet searches, they  may stumble upon older versions of these NIST documents or different documents and think they all are the same.